backend/controller/webAuthn.go

```diff
@@ -345,14 +346,22 @@ func GetYubiKeyDoorAccess(w http.ResponseWriter, r *http.Request) {
 		helpers.DBErrorHandling(res.Error, w)
 		return
 	}
-	if !credHasDoorAccess(cred, dId) {
+	hasAccess, user := credHasDoorAccess(cred, dId)
+	if !(hasAccess) {
 		http.Error(w, "User does not have access to door", http.StatusBadRequest)
 		return
 	}
+
+	err = UpdateLastTimeIn(w, user.Id)
```

Comment:
Isn't this the only place we need this logic?

Reply:
no this is when they open the door with a yubikey, since there are two flows we have two spots